core/src/main/java/org/elasticsearch/plugins/PluginCli.java

```diff
@@ -40,7 +39,20 @@ public PluginCli(Environment env) {
     }
 
     public static void main(String[] args) throws Exception {
-        BasicConfigurator.configure(new NullAppender());
+        // initialize default for es.logger.level because we will not read the logging.yml
```

Comment:
That's for providing output to the user, but the plugin install relies on Elasticsearch subsystems (the JAR hell check, for example) that do not have access to an instance of `Terminal` but do employ a logger.